invenio_opendata/testsuite/data/cms/cms-author-list-Run2011A.xml

```diff
@@ -21,6 +21,9 @@
     <datafield tag="264" ind1=" " ind2="0">
       <subfield code="c">2016</subfield>
     </datafield>
+    <datafield tag="FFT" ind1=" " ind2=" ">
+      <subfield code="a">/tmp/opendata.cern.ch-fft-file-cache/cms-authorlist-files/cms-author-list-2010.pdf</subfield>
+    </datafield>
```

Comment:
Both records have 2010 pdf, please amend the 2011 record and repush.  Otherwise LGTM.